Sources/Datadog/DatadogCore/Storage/FilesOrchestrator.swift

```diff
@@ -123,14 +131,6 @@ internal class FilesOrchestrator {
         }
     }
 
-    func deleteAllReadableFiles() {
-        do {
-            try directory.deleteAllFiles()
-        } catch {
-            DD.telemetry.error("Failed to delete all readable file", error: error)
-        }
-    }
-
```

Comment:
Rationale: this is no longer responsibility of `FilesOrchestrator` - it was moved to `FeatureStorage` instead.